src/pages/project/cartzoom.js

```diff
@@ -131,6 +130,22 @@ module.exports = {
       btnZoomOut.style.top = (boundingBox.top - (btnZoomOut.clientHeight / 2)) + 'px';
       btnZoomOut.style.left = (boundingBox.left - (btnZoomOut.clientWidth / 2)) + 'px';
     }
+  },
+
+  componentDidMount: function () {
```

Comment:
you may as well add a `removeEventListener` for good measure on `componentDidUnmount`, right?